tests/modules/images/regenerate-existing-images/background-process/perflab-background-job-test.php

```diff
@@ -0,0 +1,202 @@
+<?php
+/**
+ * Tests for background-job module.
+ *
+ * @package performance-lab
+ * @group   background-process
+ */
+
+/**
+ * Class Perflab_Background_Job_Test
+ *
+ * @coversDefaultClass Perflab_Background_Job
+ * @group background-process
+ */
+class Perflab_Background_Job_Test extends WP_UnitTestCase {
+
+	/**
+	 * Job instance.
+	 *
+	 * @var Perflab_Background_Job
+	 */
+	private $job;
+
+	public function test_class_constants_exists() {
+		$this->job = new Perflab_Background_Job();
+
+		$this->assertTrue( defined( get_class( $this->job ) . '::JOB_NAME_META_KEY' ) );
+		$this->assertTrue( defined( get_class( $this->job ) . '::JOB_DATA_META_KEY' ) );
+		$this->assertTrue( defined( get_class( $this->job ) . '::JOB_ATTEMPTS_META_KEY' ) );
+		$this->assertTrue( defined( get_class( $this->job ) . '::JOB_LOCK_META_KEY' ) );
+		$this->assertTrue( defined( get_class( $this->job ) . '::JOB_ERRORS_META_KEY' ) );
+		$this->assertTrue( defined( get_class( $this->job ) . '::JOB_STATUS_META_KEY' ) );
+		$this->assertTrue( defined( get_class( $this->job ) . '::JOB_STATUS_QUEUED' ) );
+		$this->assertTrue( defined( get_class( $this->job ) . '::JOB_STATUS_RUNNING' ) );
+		$this->assertTrue( defined( get_class( $this->job ) . '::JOB_STATUS_COMPLETE' ) );
+		$this->assertTrue( defined( get_class( $this->job ) . '::JOB_STATUS_FAILED' ) );
+	}
+
+	public function test_class_constants_values() {
+		$this->job = new Perflab_Background_Job();
+
+		$this->assertEquals( 'perflab_job_name', constant( get_class( $this->job ) . '::JOB_NAME_META_KEY' ) );
```

Comment:
Can you please create variable for `get_class( $this->job )` and use it in all place
```suggestion
		$job_class = get_class( $this->job );
		
		$this->assertEquals( 'perflab_job_name', constant( $job_class . '::JOB_NAME_META_KEY' ) );
```